priv/mssql2012.sql

```diff
@@ -642,3 +642,42 @@ CREATE TABLE muc_registered(
     nick VARCHAR(250)       NOT NULL,
     PRIMARY KEY (muc_host, luser, lserver)
 );
+
+-- from_jid, to_jid and thread have 250 characters in MySQL
```

Comment:
No, this comment is about MSSQL.
I had to reduce fields size (comparing to MySQL and Postgress) to fit into primary key size limit.

Reply:
to understand better what i mean you comment out DB creation in [mssql script](https://github.com/esl/MongooseIM/blob/612cbe8/tools/setup-mssql-database.sh#L17) called by `test-runner.sh`, and run it manually with an extra `-i` flag:
```
docker exec -it mongooseim-mssql /opt/mssql-tools/bin/sqlcmd -S localhost -U sa -P mongooseim_secret+ESL123 -d ejabberd -i mongoose.sql
```
you will notice the next warning:
```
Warning! The maximum key length for a clustered index is 900 bytes. The index 'PK_even_pusher_push_subscription' has maximum length of 1300 bytes. For some combination of large values, the insert/update operation will fail.
```
BTW this warning should be fixed!

Reply:
also we must revise if we really want have to use `NVARCHAR` type instead of `VARCHAR` for most of our mssql tables and do we really need clustered keys.
cause it seems that key for [muc_registered](https://github.com/esl/MongooseIM/blob/45c1b9c/priv/mssql2012.sql#L643) exceeds 900 bytes limit w/o any problem.

Reply:
Thanks for the explanation. Reading the comment, it was not clear to me that it is comparing with what we have in MySQL schema.